flake.nix

```diff
@@ -7,7 +7,7 @@
   };
 
   inputs = {
-    nixpkgs.url = "github:nixos/nixpkgs/nixpkgs-unstable";
+    nixpkgs.url = "github:NixOS/nixpkgs/nixos-unstable";
```

Comment:
why the switch to nixos-unstable? this is meant to be used in and outside of nixos itself, so we should be sticking with `nixpkgs-unstable` here

Reply:
I wanted to use `nixos-unstable` as the chance of a cache hit is higher for those who use the overlay

Reply:
the only cache misses i'd imagine we'd get are in some of the inputs for the linux wrapper, but it hasn't ever really been an issue in my experience. i think it might be better to just keep this for darwin users and be *slightly* more up to date (it'd also fix the current merge conflict :p)

Reply:
I get cache misses for Prism Launcher regularly and if this repo is mostly ahead of `nixos-unstable`, I am sure most other NixOS users will too. Also, there is no reason for macOS users to avoid `nixos-unstable` either.

Reply:
> I get cache misses for Prism Launcher (-unwrapped) regularly and if this repo is mostly ahead of nixos-unstable, I am sure most other NixOS users will too

if you mean misses for the (un)wrapped packages themselves in our cache, i think that's to be expected when using an overlay and not something we should really go out of our way to fix (as it may only really "fix" this issue for a select few who happen to update at a similar place to our flake.lock and use unstable)

> Also, there is no reason for macOS users to avoid nixos-unstable either.

build fixes for macos are usually fast tracked to nixpkgs-unstable a good bit before nixos-unstable, which isn't a reason to *avoid* nixos-unstable per se, but a reason to not use it

and regarding cache misses with our overlay again: using the nixos branch would basically always put us behind a good chunk of darwin users, who will probably run into more misses as they get updates faster. regardless of if we make this change or keep the current nixpkgs-unstable branch, nixos or macos users will be getting the short hand of the stick here. i don't see much point in changing this when that's the case and it doesn't even really fix the issue across the board. if people want to use our binary cache consistently, it'd be best to use the packages from the flake directly

if you really think this is worth it though, i'll approve. this has been open long enough 😆